model/src/main/kotlin/PackageCurationData.kt

```diff
@@ -45,8 +45,9 @@ data class PackageCurationData(
     val declaredLicenses: SortedSet<String>? = null,
 
     /**
-     * The concluded license as an [SpdxExpression]. It can be used to correct the license of a package in case the
-     * [declaredLicenses] found in the packages metadata or the licenses detected by a scanner do not match reality.
+     * The concluded license as an [SpdxExpression]. It can be used to correct the
+     * [declared licenses of a package][Package.declaredLicenses] in case the found in the packages metadata or the
```

Comment:
Nit: You can have a line-break between `][` (exactly as in my suggested code for copy & paste) without breaking the link. That makes the formatting a bit nicer.